<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FhiAimsInputDialog</class>
 <widget class="QDialog" name="FhiAimsInputDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>765</width>
    <height>680</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>100</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Fhi-aims Input</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="2" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>575</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="controlTab">
        <attribute name="title">
         <string>Control</string>
        </attribute>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>-7</y>
           <width>721</width>
           <height>231</height>
          </rect>
         </property>
         <property name="title">
          <string>Physical Model</string>
         </property>
         <widget class="QWidget" name="gridLayoutWidget">
          <property name="geometry">
           <rect>
            <x>9</x>
            <y>29</y>
            <width>461</width>
            <height>131</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="1">
            <widget class="QComboBox" name="xcCombo">
             <item>
              <property name="text">
               <string>pw-lda</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>pz-lda</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>vwn</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>vwn-gauss</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>am05</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>blyp</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>pbe</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>pbeint</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>pbesol</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>rpbe</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>revpbe</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>b3lyp</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>hse03</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>hse06</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>pbe0</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>pbeso10</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>hf</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>mp2</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>screx</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>cohsex</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>pbe_vdw</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>revpbe_vdw</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="xcLabel">
             <property name="text">
              <string>XC Functional</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="spinPolCombo">
             <item>
              <property name="text">
               <string>None</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Collinear</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="spinLabel">
             <property name="text">
              <string>Spin-polarization</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="relativityCombo">
             <item>
              <property name="text">
               <string>None</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="relativityLabel">
             <property name="text">
              <string>Relativistic</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="chargeLabel">
             <property name="text">
              <string>Charge</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QDoubleSpinBox" name="chargeSpin"/>
           </item>
           <item row="1" column="2">
            <widget class="QLabel" name="momentLabel">
             <property name="text">
              <string>Initial Moment</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QDoubleSpinBox" name="momentSpin">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QTextEdit" name="previewTextControl">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>230</y>
           <width>721</width>
           <height>311</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>300</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Monaco</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
         <property name="acceptRichText">
          <bool>false</bool>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="geomTab">
        <attribute name="title">
         <string>Geometry</string>
        </attribute>
        <widget class="QWidget" name="horizontalLayoutWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>721</width>
           <height>31</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Geometry Specification</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="crystalCombo">
            <item>
             <property name="text">
              <string>Use Displayed Geometry</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Generate Primitve Lattice and Basis</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QTextEdit" name="previewTextGeometry">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>50</y>
           <width>721</width>
           <height>491</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>300</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Monaco</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
         <property name="acceptRichText">
          <bool>false</bool>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="resetButton">
         <property name="text">
          <string>Reset</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="enableFormButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Use Form</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="moreButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Hide Preview</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>216</width>
           <height>26</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="generateButton">
         <property name="text">
          <string>Generate...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="closeButton">
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QLineEdit" name="titleLine">
     <property name="text">
      <string>Title</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
